pkg/network/node/pod.go

```diff
@@ -160,13 +161,39 @@ func (m *podManager) Start(rundir string, localSubnetCIDR string, clusterNetwork
 	return m.cniServer.Start(m.handleCNIRequest)
 }
 
+func (m *podManager) InitRunningPods(existingPodSandboxes map[string]*kruntimeapi.PodSandbox, existingOFPodNetworks map[string]podNetworkInfo) error {
+	m.runningPodsLock.Lock()
+	defer m.runningPodsLock.Unlock()
+	defer glog.V(5).Infof("Finished initializing podManager with running pods at start-up")
+
+	for _, sandbox := range existingPodSandboxes {
+
+		sKey := getPodKey(sandbox.Metadata.Namespace, sandbox.Metadata.Name)
+
+		vnid, err := m.policy.GetVNID(sandbox.Metadata.Namespace)
+		if err != nil {
+			return err
+		}
+
+		podNetworkInfo, ok := existingOFPodNetworks[sandbox.Id]
+		if !ok {
+			glog.Errorf("Inexistant openflow network for sandbox pod with ID: %s", sandbox.Id)
```

Comment:
"Nonexistent". Also "openflow network" isn't really the right term. I'd say something like `"No network information for pod %s/%s", sandbox.Metadata.Namespace, sandbox.Metadata.Name`. Also, warning not error.

Reply:
If this happens, we have a sandbox pod without any `podNetworkInfo`, meaning we won't be able initialize `runningPods` correctly, which will probably lead to errors (possibly subtle and hard-to-debug ones). Are you sure that this should be logged as a warning solely?